README.md

```diff
@@ -56,10 +56,16 @@ With PASS we are looking into using RDF to link between pods and store metadata
 
 | Development Team | UX/UI Team | Project Management Team |
 | ---------------- | ---------- | ----------------------- |
-| Jared K.         | Gabby      | Flo                     |
-| Kevin M.         | Kyle       | Danica                  |
-| Ka Hung          |            | Wilfred                 |
-| Greg Witt        |            | Katharine               |
+| Jared K.         | Gabby P.   | Flo K.                  |
+| Kevin M.         | Kyle B.    | Danica B.               |
+| Ka Hung L.       | Laura U.   | Wilfred P.              |
+| Greg W.          | Shelby P.  | Katharine               |
+| Emily S.         | Andy W.    |                         |
+| Ben C.           | Scott B.   |                         |
+| Tim S.           | Nicholas   |                         |
+| Brian H.         |            |                         |
+| Georgia          |            |                         |
```

Comment:
Not an issue, but any reason why Georgia, Nicholas, and Katharine are missing their initials?

Reply:
I don't know their surnames. XD